boards/arm/quick_feather/Kconfig.defconfig

```
@@ -11,7 +11,7 @@ config BOARD
 if PWM
 
 config PWM_LITEX
-	select SOC_EOS_S3_FPGA
+	select EOS_S3_PROGRAM_FPGA
 	default y
 
```

If you want to use your own driver instead of `PWM_LITEX` please add proper configs here like:
```suggestion
config PWM_EOS_S3
    select SOC_EOS_S3_FPGA
    select EOS_S3_PROGRAM_FPGA
    default y
```
and then disable `PWM_LITEX` in `prj.conf` in sample: `CONFIG_PWM_LITEX=n`

---

Earlier the FPGA was loaded through JLink. As part of eos_s3_init() FPGA clocks used to get initialized. Instead of this now the FPGA bit stream is loaded to EOS S3. Clock init happens as part of it. 
In both cases we have been using our FPGA IP. There is no change in the flow. That's why I have not created a new config. If you still thing I have to go with new config, I'll do. Please let me know.